dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilder.java

```diff
@@ -262,22 +264,20 @@ public static UnresolvableArtifactProblem createUnresolvableArtifactProblem(
    *
    * @param firstNode node to start traversal
    */
-  private DependencyGraph levelOrder(DependencyNode firstNode) {
+  public static DependencyGraph levelOrder(DependencyNode firstNode) {
```

Comment:
LinkageCheckerRule uses this to convert Maven's DependencyNode to our DependencyGraph.